hudi-common/src/main/java/org/apache/hudi/common/config/HoodieReaderConfig.java

```diff
@@ -89,4 +91,15 @@ public class HoodieReaderConfig extends HoodieConfig {
       "hoodie.write.record.merge.custom.implementation.classes";
   public static final String RECORD_MERGE_IMPL_CLASSES_DEPRECATED_WRITE_CONFIG_KEY =
       "hoodie.datasource.write.record.merger.impls";
+
+  public static final ConfigProperty<Double> LOG_FILE_TO_PARQUET_FORMAT_SIZE_ESTIMATION_FRACTION = ConfigProperty
+      .key("hoodie.logfile.to.parquet.format.size.estimation.fraction")
```

Comment:
Why not just use `config.getLogFileToParquetCompressionRatio()`?

Reply:
> Why not just use `config.getLogFileToParquetCompressionRatio()`?

`HoodieStorageConfig#LOGFILE_TO_PARQUET_COMPRESSION_RATIO_FRACTION` belongs to the group `WRITE_CLIENT`.
<img width="543" alt="image" src="https://github.com/user-attachments/assets/1342b79a-2094-4f9d-9dc7-1fe94687a99a" />

Is it unreasonable for us to use a write client-related parameter when reading? By the way, this also raises another issue, many of the parameters that we use should be common to both read and write, but we put them in write config

Reply:
> Is it unreasonable for us to use a write client-related parameter when reading?

the write config is shared with the reader.

Reply:
> the write config is shared with the reader.

Done~